easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20230930-R2-intel-compilers-2022.1.0.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2017 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+#
+# Authors::
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Ward Poelmans <wpoely86@gmail.com>
+# * Oliver Stueker <oliver.stueker@ace-net.ca>
+# License::   MIT/GPL
+##
+name = 'GAMESS-US'
+version = '20230930-R2'
+
+ddi_comm = 'sockets'
+maxcpus = '1024'
+maxnodes = '1'
+hyperthreading = False
+
+homepage = 'https://www.msg.chem.iastate.edu/gamess/'
+description = """The General Atomic and Molecular Electronic Structure System (GAMESS)
+is a general ab initio quantum chemistry package.
+This module can be used on a maximum of %s nodes and %s CPUs per node.
+ """ % (maxnodes, maxcpus)
+
+toolchain = {'name': 'intel-compilers', 'version': '2022.1.0'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('imkl', '2022.1.0', '', SYSTEM),
+]
+
+# manually download via http://www.msg.chem.iastate.edu/gamess/download.html (requires registration)
+# rename gamess-current.tar.gz by changing 'current' to the proper version
```

Comment:
use `download_instructions` for this

Reply:
good point, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19452/commits/8b3487586eeb4759bcd46ac5381aacfb15a6cca2